app/assets/stylesheets/active_admin/components/_comments.scss

```diff
@@ -1,41 +1,25 @@
-// -------------------------------------- Admin Notes
-.comments {
+.active_admin_comment {
+  margin-bottom: 25px;
+  max-width: 45em;
+}
 
-  .active_admin_comment {
-    clear: both;
-    margin-top: 10px;
-    margin-bottom: 40px;
-    max-width: 700px;
+.active_admin_comment_meta {
```

Comment:
Selector `active_admin_comment_meta` should be written in lowercase with hyphens